java/client/src/main/java/glide/api/models/commands/ScoreModifier.java

```diff
@@ -0,0 +1,17 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.SortedSetBaseCommands;
+
+// TODO add @link to ZMPOP when implemented
+/**
+ * Mandatory option for {@link SortedSetBaseCommands#bzmpop(String[], ScoreModifier, double)} and
+ * for {@link SortedSetBaseCommands#bzmpop(String[], ScoreModifier, double, long)}. Defines which
+ * elements to pop from the sorted set.
+ */
+public enum ScoreModifier {
```

Comment:
why ScoreModifier?  This doesn't really modify the score, but is more like a filter.  Maybe `ScoreFilter`?

Reply:
Name copied from docs. Will rename, thanks.